Phat-Taupe-Pests/BangazonAPI
# Added Cors policy and cors-test docs

## DESCRIPTION

Added the CORS Policy So That Only Bangazonians Are Allowed Access

## STEPS TO TEST
1. Clone this branch to a local repo
2. In terminal, run `dotnet ef migrations add {migrationName}`, `dotnet ef database update`, `dotnet build` then `dotnet run`
3. Test functionality
	A. Open a second terminal and cd into the `cors-test` folder
	B. Start a new local server 
	C. Go to local host in a browser and open your developer tools
        D. Check the console, It should display CORS Working Request Failed

## REFERENCE

[Ticket](https://github.com/Phat-Taupe-Pests/BangazonAPI/issues/12)

[ERD](https://github.com/Phat-Taupe-Pests/Bangazon)
